diff --git a/freqtrade/wallets.py b/freqtrade/wallets.py
index e644d82..65e3077 100644
--- a/freqtrade/wallets.py
+++ b/freqtrade/wallets.py
@@ -304,6 +304,14 @@ class Wallets:
         or by using current balance subtracting
         """
         if "available_capital" in self._config:
+            if (
+                "tradable_balance_ratio" in self._config
+                and self._config["tradable_balance_ratio"] != 0.99
+            ):
+                logger.warning(
+                    "You have set both `available_capital` and `tradable_balance_ratio` - "
+                    "only `available_capital` will be used, `tradable_balance_ratio` is ignored."
+                )
             return self._config["available_capital"]
         else:
             tot_profit = Trade.get_total_closed_profit()
